circleci/images/pgupgradetester/files/sbin/install-testdeps

```diff
@@ -47,16 +47,11 @@ echo "deb http://apt.postgresql.org/pub/repos/apt/ ${codename}-pgdg main ${pg_la
 pkg_cloud_url="https://packagecloud.io/install/repositories/citus-bot/the-process/script.deb.sh"
 curl -sf "${pkg_cloud_url}" | unique_id=exttester bash
 
-cat > /etc/apt/preferences.d/citusdata_the-process.pref <<EOF
-Package: postgresql-server-dev-*
-Pin: release o=packagecloud.io/citus-bot/the-process
-Pin-Priority: 700
-EOF
 
 echo "Installing tools for testing PostgreSQL extensions..." >&2
 sd_pkgs=( "${pg_majors[@]/#/postgresql-server-dev-}" )
 pg_pkgs=( "${pg_majors[@]/#/postgresql-}" )
-apt-get update && apt-get install -y --no-install-recommends make postgresql-common perl libcurl3 "${sd_pkgs[@]}" "${pg_pkgs[@]}"
```

Comment:
why do we need to remove the `--no-install-recommends`? Is this due to the ldap packages we added during the build?

Reply:
I think I was trying something and forgot this, reverted back.